### DISCUSSION TITLE
Character is freezed when bombs or grenades are used

### DISCUSSION THREAD

**Discussion by GinoX27** [Pryaxis/TShock#1860] [Q&A]
Title.

I don't know how to describe the issue better than that.

It doesn't happen as superadmin

Is it known? Is there a fix?

**Comment by grechnevi** [comment]
> Character is freezed when bombs or grenades are used 

Use _tshock.ignore.removetile_ permission to remove this restriction.
Default group for common players is _default_.
/group addperm default tshock.ignore.removetile

Permission list:
https://tshock.readme.io/docs/permissions

**Reply by GinoX27** [reply]
Not working. This permission is already present on guest (I'm playing with trusted friends) and they are still getting freezed

**Comment by hakusaro** [comment]
@GinoX27 can you try on the latest pre-release (7), and set `DebugLogs` to `true` in `config.json` and then send the logs when friends use them? Are these the wet bombs, honey bombs, dirt bombs, or lava bombs?
